README.md

```diff
@@ -1,16 +1,189 @@
+accepts an argument of a person's name, we would do it like this:
+
+```ruby
+    #method name      #argument
+def greeting_a_person(name)
+  "Hello #{name}"
+end
+```
+
+Arguments create new _local_ variables that can be used within the method. When
+you name an argument, you are defining what word you want to use to access that
+data, just like when you create a variable. Arguments follow the same rules as
+local variables: they can be any word that starts with a lowercase letter and
+they should be as descriptive of the data as possible.
+
+In our `#greeting_a_person` method example, we are saying: When you call the
+`#greeting_a_person` method with an argument of `"Maria"`, set a variable `name`
+equal to the value of `"Maria"`.
+
+Let's open up IRB and try this out! First, take the `#greeting_a_person` code
+sample and paste it into your command line. Once that has been entered, you can
+test out the method like this:
+
+```bash
+greeting_a_person("Maria")
+# => "Hello Maria"
+```
+or this:
+
+```bash
+name = "Maria"
+greeting_a_person(name)
+# => "Hello Maria"
+```
+
+This same pattern works for passing in as many arguments as you want by adding a
+comma (`,`) between each argument you want to pass in. Let's try creating a
+method that accepts two arguments: a person's name and their programming
+language of choice.
+
+```ruby
+  # method name      first_argument, second_argument
+def greeting_programmer(name, language)
+  puts "Hello, #{name}. We heard you are a great #{language} programmer."
+end
+
+greeting_programmer("Maria", "Ruby")
+# > Hello, Maria. We heard you are a great Ruby programmer.
+
+greeting_programmer("Steven", "Elixir")
+# > Hello, Steven. We heard you are a great Elixir programmer.
+```
+
+Once you define arguments for a method, they become _required_ when you invoke,
+or call, the method. What happens when you don't pass in the required argument?
+When an argument with a value is not passed int, you get an `ArgumentError`.
+Here's an example:
+
+```ruby
+def greeting(name)
+  puts "Hello, #{name}!"
+end
+
+greeting # We call the method without a value for the argument `name`
+# > ArgumentError: wrong number of arguments (0 for 1)
+
+greeting(name) # If we call the method without passing in a value for the variable `name` we see:
+# > NameError: undefined local variable or method `name' for main:Object
+```
+
+The word, in this case `name`, that we use as the argument's name in the method
+signature becomes a local variable within the method. Through that variable we
+can reference the value of the argument inside the body of the method.
+
+Additionally, a method defined to accept one argument will raise an error if
+called with more than one argument.
+
+```ruby
+def greeting(name)
+  puts "Hello, #{name}!"
+end
+
+greeting("Maria", "Ruby") # The method accepts 1 argument and I supplied 2.
+# > ArgumentError: wrong number of arguments (2 for 1)
+```
+
```

Comment:
Maybe add: 

Again, the _arity_ of greeting is one, by providing it _two_ arguments, Ruby gets upset.